source/envs/e2048.py

```
@@ -0,0 +1,317 @@
+
+gym.register_envs(ale_py)
+# from stable_baselines3 import PPO
+from stable_baselines3.common.env_util import make_vec_env
+from source.envs.env import Env
+
+from source.envs.env2048 import Env2048PG, _autoplay_step
+from source.env_callback.save_on_step_callback import SaveOnStepCallback
+from stable_baselines3.common.atari_wrappers import AtariWrapper
+from stable_baselines3.common.vec_env import VecFrameStack, DummyVecEnv
+from stable_baselines3.common.vec_env import VecNormalize, SubprocVecEnv
+from stable_baselines3.common.env_util import make_vec_env
```

Ruff (0.12.2)

Redefinition of unused `make_vec_env` from line 18

Remove definition: `make_vec_env`

(F811)